doxia-core/src/main/java/org/apache/maven/doxia/markup/HtmlMarkup.java

```diff
@@ -473,6 +473,17 @@ public String toString()
     /** Xhtml tag for <code>kbd</code>. */
     Tag KBD = Tag.KBD;
 
+    /** Html5 tag for <code>keygen</code>. */
```

Comment:
Html5 --> HTML 5

Reply:
It is actually HTML5. See https://html.spec.whatwg.org/multipage/introduction.html#is-this-html5?